backend.pmd-ruleset.xml

```diff
@@ -1,22 +1,21 @@
-        <!--
```

Comment:
I don't know why this particular file contains license info. I'd remove it

Reply:
Removed this.